package.json

```diff
@@ -102,9 +103,8 @@
     },
     "lint-staged": {
         "*.{ts, tsx,js,jsx}": [
-            "pretty-quick --staged",
-            "git add",
-            "yarn lint:ts"
+            "prettier --write",
+            "git add"
```

Comment:
I think we should add eslint --fix here.

We did discuss tsc, but we have added in the pr-template that contributors should make sure yarn build run before opening a PR, and that will make sure that type-error will get redlighted. So lets not do that on commit.

Reply:
`eslint --fix` can do dangerous code changes (an example is the react hooks eslint plugin that also has fix for warnings). I added eslint instead.

Reply:
why eslint?! you were supposed to help us! :crying_cat_face: